ui-modules/blueprint-composer/app/components/providers/blueprint-service.provider.js

```diff
@@ -230,7 +230,7 @@ function BlueprintService($log, $q, $sce, paletteApi, iconGenerator, dslService,
     }
 
     function lookup(entity, id, any = false) {
-        if (entity._id === id) {
+        if ((entity._id === id) || (entity.id === id)) {
```

Comment:
Do we still need this? I thought we agreed to not engage public ID of the Entity (which is optional, can be null, and not unique) into lookup.